scripts/ci_before_init_embed.bash

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-export WEBOTS_VERSION=2022a
+export WEBOTS_RELEASE_VERSION=2022a
```

Comment:
Shouldn't it be 2022b here?
```suggestion
export WEBOTS_RELEASE_VERSION=2022b
```

Reply:
Yes I wanted to do it in another PR but it is better to do it in this one. Thanks.